release/models/system/openconfig-aaa-radius.yang

```diff
@@ -26,7 +26,13 @@ submodule openconfig-aaa-radius {
     related to the RADIUS protocol for authentication,
     authorization, and accounting.";
 
-  oc-ext:openconfig-version "0.5.0";
+  oc-ext:openconfig-version "0.6.0";
```

Comment:
Because this is a breaking change (modifying an existing leaf) we should bump the version to 1.0.0 in all these files